app/controllers/users_controller.rb

```diff
@@ -1,13 +1,24 @@
 class UsersController < ApplicationController
   before_action :set_user, only: %i[ show liked feed followers following discover ]
+  before_action only: [:index, :edit, :update, :destroy, :show, :feed] do
```

Comment:
missing discover and liked, also we don't have index, edit, update, destroy. You can remove the `only: ...` statement altogether so it applies to all actions